packages/documentation/src/layouts/layout.js

```diff
@@ -135,6 +135,14 @@ class Layout extends React.Component {
                 content:
                     'Template for creating design system documentatation',
               },
+              {
```

Comment:
So these are just examples of meta tags we can put on pages. This part of the documentation suggests to me that we can use the meta tags to influence the way the page is indexed in the search. 

I'm putting this in the layout for now but wondering how we would organize this for page specific content. Would we have the layout query metadata (not sure what props are available on `data.site`) from the child document or push this `<Helmet>` down into the children?
![screen shot 2019-03-06 at 08 15 59](https://user-images.githubusercontent.com/4998130/53891675-1b084480-3fe8-11e9-885e-a22c22e1706b.png)